dataservices/dataservices.go

```diff
@@ -175,6 +176,30 @@ func process(iter store.Iterator) data.Dataset {
 	return all
 }
 
+func buildQuery(params url.Values) store.Query {
+
+	types := strings.Split(params.Get("type"), ",")
```

Comment:
Too bad we don't have some sort of Compact on the array to remove nils and whitespace-only and empty strings.